spyder/widgets/ipythonconsole/client.py

```diff
@@ -297,6 +297,9 @@ def get_name(self):
             # Adding id to name
             client_id = self.id_['int_id'] + u'/' + self.id_['str_id']
             name = name + u' ' + client_id
+        elif self.given_name in ["PyLab", "SymPy"]:

```

Comment:
This is a little silly, but it works.  The down side is that you can't rename a tab to "PyLab" or "SymPy" but that's a corner case.